Update README.md for clarity and concision

b1skit opened the pull request:
Reworded several sentences and paragraphs for clarity and concision. Corrected several spelling and grammatical errors. All original information was maintained.

All pull requests must have an associated issue in the issue tracker. If there
isn't one, please go open an issue describing the defect, deficiency or desired
feature. You can read more about our issue and PR processes in the
[wiki](https://github.com/bokeh/bokeh/wiki/BEP-1:-Issues-and-PRs-management).

- [x] issues: fixes #7690
- [ ] tests added / passed
- [ ] release document entry (if new feature or API change)

bryevdv commented on README.md:
```diff
@@ -255,27 +254,27 @@ streaming datasets.
-This installs Bokeh and all needed dependencies.
-
-To begin using Bokeh or to install using `pip`, follow the [Quickstart](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html) documentation.
+The Bokeh [quickstart documentation](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html) provides information on installing Bokeh using [`pip`](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html#quick-installation). It also includes a [getting started](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html#getting-started) guide.
```
"pip" wil make a very small visual target for users to click. Also for parallelism I think this phrase should begin with another appositive to mirror the "to install" above. Also, the the "getting started" section is useful regardless of the method of installation. Here is seems like it only applies to the pip instructions. Perhaps:

> To install using `pip`, enter the following command at a Bash or Windows command prompt:
> ```
> pip install bokeh
> ```
>
> For more information, refer to the [installation documentation](https://bokeh.pydata.org/en/latest/docs/installation.html).
> 
> Once Bokeh is installed, check out the [Getting Started](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html#getting-started) section of the [Quickstart](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html) .

bryevdv commented on README.md:
```diff
@@ -255,27 +254,27 @@ streaming datasets.
 Installation
 ------------
-We recommend using the [Anaconda Python distribution](https://www.anaconda.com/what-is-anaconda/) and conda to install Bokeh. Enter this command at a Bash or Windows command prompt:
+The easiest way to install Bokeh is using the [Anaconda Python distribution](https://www.anaconda.com/what-is-anaconda/) and its included *Conda* package management system. This allows Bokeh and its required dependencies to be installed by entering the following command at a Bash or Windows command prompt:
```
This seems more wordy and complicated to me. Again I prefer imperative call to action: *To do task X, perform action Y*.

b1skit commented:
Hi bryevdv, thanks for your feedback. I agree with your points, I'll make some more changes.

b1skit commented:
@bryevdv Sorry for the spam, a bit of a n00b question here: I still see "1 review requesting changes." I'm pretty sure I addressed all of your comments. Is there something I missed or that need to click to pass this back to you?

Thanks,

bryevdv commented:
No it's just up to a maintainer to review again, I will look at things tommorow, it's late :)

bryevdv commented:
@b1skit I've approved the changes. There was a spurious test failure, so I have restarted the CI  build and will merge later today when everything is green. Thanks for the PR!
